app/models/manageiq/providers/autosde/storage_manager/ems_refresh_workflow.rb

```diff
@@ -31,7 +31,7 @@ def poll_native_task
   def poll_refresh_task
     opts = {
       :query_params => {
-        :object_id => options[:native_object_id]
+        :object_id => options[:native_objects_ids][0]
```

Comment:
Hm this looks weird, do you want to wait until all volumes have finished instead of just the first one?

Reply:
The method poll_refresh_task is only intended to deal with the case where we have to wait for a second task to success. This only happens when we add a physical storage (see lines 16 - 19) and then we have to wait for the autosde refresh (that's the second task) to succeed in order to do a general refresh.

In that case, there is only one object_id in the task.

If we add a bulk of volumes, all volumes will have finished their creation until the status of the task is success, in line 13.